lib/scriptinterpreter.js

```diff
@@ -3,6 +3,7 @@ var bignum = require('bigi');
 var Script = require('btc-script');
 var cryptoHash = require('crypto-hashing')
 var ecdsa = require('ecdsa')
+var Transaction = require('btc-transaction').Transaction
```

Comment:
I wouldn't want to pull the entire of bitcoinjs-lib as a dependency. Besides, btc-transaction is tested and is being used in production (by myself and others), I don't see why we should replace it